style.css

```
@@ -6,16 +6,18 @@
 #DAFDBA
 */
 
-.rivertitle { /* Titles on the catalog page */
+.rivertitle {
+  /* Titles on the catalog page */
   padding: 0.5em;
   background-color: #012030;
 }
 
-.riversummary { /* Blurbs on the catalog page */
+.riversummary {
+  /* Blurbs on the catalog page */
   padding-left: 0.75em;
 }
 
-a:hover { /* Links that are being hovered */
+li:hover { /* List items that are being hovered */
```

This reverts commit af9351149a490d02bd19e2c6a3e799eb2bcc2e30 . I don't think that is desirable behavior (leads to weird highlighting on the References page).

---

With the changes to the nav bar li:hover seems to work fine. I would like to keep it this way unless there's still an issue on your end and if there is then we should probably use a css class to solve it because the anchor tag is used for a lot of things and the nav bar should have it's own unique highlighting.

---

In the nav bar itself, li:hover works great. But on the References page (or anywhere else we might want a bulleted list), it _also_ highlights that. Example of what I mean: [https://imgur.com/a/u7vTlRq](https://imgur.com/a/u7vTlRq).

```suggestion
nav ul li:hover { /* List items in the nav bar that are being hovered */
```

---

Nice! That's a good idea.